ynput/OpenPype
# Houdini: Extract active view as thumbnail when no thumbnail set

## Changelog Description
Add a function to take snapshoots from scene view in Houdini.
It works in different Houdini context e.g. Objects, Solaris.
For what it's worth, this function allows creating thumbnails image sequence by specifying `fstart` and `fend`. 

## Additional info
I think this functionality can be used to create thumbnails on the go when publishing.

## Preview
![image](https://github.com/ynput/OpenPype/assets/20871534/644a3aa9-e953-4189-a147-1caab2083f6b)

## Testing notes:
1. Launch Houdini via Openpype or Ayon Launcher.
2. publish workfile
3. published workfiles should have a thumbnail, you will be able to see that in loader.
